plugins/node/opentelemetry-instrumentation-aws-lambda/src/instrumentation.ts

```diff
@@ -396,10 +376,32 @@ export class AwsLambdaInstrumentation extends InstrumentationBase {
     if (trace.getSpan(extractedContext)?.spanContext()) {
       return extractedContext;
     }
-    if (!parent) {
-      // No context in Lambda environment or HTTP headers.
-      return ROOT_CONTEXT;
+    // No context in Lambda environment or HTTP headers.
+    return ROOT_CONTEXT;
+  }
+
+  private static _determineLinks(): Link[] {
```

Comment:
@willarmiros to be clear since you are the component owner, are you asking that we block this change? Since this comes from the spec, I would lean on the side of accepting it. Also, since this component is `0.x` version it should be expected that breaking changes are possible.